### PR TITLE
chore: add cypress tsconfig

### PR DESCRIPTION
**Pull request by MuckT**
### Description 
Cypress Dapp Up Check started failing.
Failing Run: https://github.com/valora-inc/dapp-list/actions/runs/4917657772/jobs/8783021884
Fixed Passing Run: https://github.com/valora-inc/dapp-list/actions/runs/4918889815/jobs/8785806024

This should fix that.

### CODE REVIEW

File: cypress/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../tsconfig.json",
+  "include": ["../node_modules/cypress", "*/*.ts"],
```
Comment (silasbw) [inline cypress/tsconfig.json:3]: surprised by the node_modules inclusion.  Is that for sourcemaps or something?

Comment (MuckT) [reply cypress/tsconfig.json:3]: I am not entirely sure, I do see it like this in a lot of other [repos](https://github.com/search?q=%22include%22%3A+%5B%22..%2Fnode_modules%2Fcypress%22%2C+%22*%2F*.ts%22%5D%2C&type=code).

Comment (MuckT) [reply cypress/tsconfig.json:3]: I also see it without in several [repos](https://github.com/search?q=%22include%22%3A+%5B%22*%2F*.ts%22%5D&type=code).

Comment (MuckT) [reply cypress/tsconfig.json:3]: I'll remove and check that CI still passes.

Comment (MuckT) [reply cypress/tsconfig.json]: @silasbw stripped down to just what we need in a4ef02554e36f189a38d382dd410fa62b60aed94!

**Review by kathaypacific — APPROVED** [review]

File: cypress/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../tsconfig.json",
+  "include": ["*/*.ts"],
```
Comment (jeanregisser) [inline cypress/tsconfig.json:3]: nit: would it work like this too?
```suggestion
  "include": ["."],
```

Comment (MuckT) [reply cypress/tsconfig.json]: Working correctly with: `["."]`: [action run](https://github.com/valora-inc/dapp-list/actions/runs/4926526585/jobs/8802202182)!

**Pull request merged**